### ISSUE TITLE
Slabs/Ice/Trapdoors movement glitch (bug)

### ISSUE THREAD

**Issue by reded1** [NoCheatPlus/Issues#350]
Hello.
I'd want to report a weird thing that has happened to me while running and jumping on any of the blocks in the title, while having another block up to me. It sets me back
It's not really important, but I don't think it would be hard to patch ;)

**Comment by asofold** [comment]
What do you mean by 'another block up to me'? I can't make out what the map looks like there.

And could you copy and paste the complete output of the 'ncp version' command to the end of the description? That would help to speed up things a lot (versions, some setup info ...).

**Comment by jonesdj1** [comment]
@asofold if you sprint and jump on ice in a 1.9 highway with packet ice on the ground if you spam space fast you will flag survival_fly now this has been a thing for a long time too.

**Comment by asofold** [comment]
So this is less than 2 blocks ceiling height?

**Comment by jonesdj1** [comment]
yeah just a little less it also happens in 2 high ceilings too just not as bad

**Comment by RoboMWM** [comment]
While testing this out with /ncp debug (which I assume disables setbacks on the debugged player), I was still rubberbanded despite no messages appearing in server console. I didn't run anything monitoring teleport events, but I'm going to guess that this one is vanilla-caused.

**Comment by 0-x-2-2** [comment]
Definitely not vanilla caused if you install [testncp](https://dev.bukkit.org/projects/ncptools) on the server you will be flagging survival_fly.

**Comment by RoboMWM** [comment]
I'm being rubberbanded while having debug mode on which iirc disables setbacks. Don't see how it matters if NCP is flagging it if vanilla is interfering regardless.

**Comment by 0-x-2-2** [comment]
Only happens with ncp installed so still a ncp issue regardless

**Comment by RoboMWM** [comment]
Just tested on a vanilla server, and was being set back.

**Comment by asofold** [comment]
* Vanilla set back could also trigger subsequent survivalfly :) on certain occasions.
* NCP debugging doesn't disable set-backs (!).

Likely this is just NCP blocking it, because the air-to-ground distance is even lower than with a 2-high ceiling. My test setup doesn't have trap doors either.
